docs/podman-ps.1.md

```diff
@@ -112,6 +112,11 @@ In some cases, a container's state in the runtime can become out of sync with Po
 This will update Podman's state based on what the OCI runtime reports.
 Forcibly syncing is much slower, but can resolve inconsistent state issues.
 
+**--watch**
+
+Allows you to follow the output of the ps command on an interval in seconds.  It is very similar
```

Comment:
Suggest rephrase to `Refresh the output with current containers on an interval in seconds. An interval of 0 seconds displays only once.`

Reply:
ill take sentence one... i dont want to support an interval of 0; watch needs to be a positive integer.